app/src/main/java/org/mozilla/focus/appreview/AppReviewUtils.kt

```diff
@@ -100,15 +100,20 @@ class AppReviewUtils {
             )
 
             return inAppReviewStep == AppReviewStep.ReviewNeeded.name || (
-                lastReviewedTime +
-                    APP_REVIEW_TIME_TRIGGER <= System.currentTimeMillis() &&
-                    inAppReviewStep == AppReviewStep.Reviewed.name
-                )
+                    lastReviewedTime +
```

Comment:
weird, I haven't changed the indentation here, and locally the dif does not show this.